test_commands.sh

```diff
@@ -135,50 +135,44 @@ install_package_from_source() {
 }
 
 report() {
-    if [ "$_arg_verbose" = "on" ]; then
-        if [ $test_result -eq 0 ]; then
-            printf "Result: %-70s \x1b[32mpassed\x1b[0m\n" "$*"
-        else
-            printf "Result: %-70s \x1b[31mfailed\x1b[0m\n\n" "$*"
-        fi
+    if [ $test_result -eq 0 ]; then
+        printf "Result: %-70s \x1b[32mpassed\x1b[0m\n" "$*"
```

Comment:
Argbash will complain about unbalanced square brackets, which can be fixed by a comment:

```suggestion
        printf "Result: %-70s \x1b[32mpassed\x1b[0m\n" "$*"  # ]]
```